src/FlaUI.Core/Elements/Tab.cs

```diff
@@ -5,49 +5,49 @@
 
 namespace FlaUI.Core.Elements
 {
-    public class Tab : Element
+    public class Tab : AutomationElement
     {
         public Tab(AutomationObjectBase automationObject) : base(automationObject)
         {
         }
 
         /// <summary>
-        /// The currently selected <see cref="TabItem"/>
+        /// The currently selected <see cref="TabItemAutomation"/>
         /// </summary>
-        public TabItem SelectedTabItem
+        public TabItemAutomation SelectedTabItemAutomation
         {
-            get { return TabItems.FirstOrDefault(t => t.IsSelected); }
+            get { return TabItemsAutomation.FirstOrDefault(t => t.IsSelected); }
         }
 
         /// <summary>
-        /// The index of the currently selected <see cref="TabItem"/>
+        /// The index of the currently selected <see cref="TabItemAutomation"/>
         /// </summary>
         public int SelectedTabItemIndex => GetIndexOfSelectedTabItem();
 
         /// <summary>
-        /// All <see cref="TabItem"/> objects from this <see cref="Tab"/>
+        /// All <see cref="TabItemAutomation"/> objects from this <see cref="Tab"/>
         /// </summary>
-        public TabItem[] TabItems => GetTabItems();
+        public TabItemAutomation[] TabItemsAutomation => GetTabItems();
```

Comment:
Switched all TabItemAutomation and TabItemsAutomation back to TabItem and TabItems respectively. Resharper renamed this and I didn't notice.